pkg/pillar/types/downloadertypes.go

```diff
@@ -167,3 +168,67 @@ func AllowNonFreePort(gc ConfigItemValueMap, objType string) bool {
 		return false
 	}
 }
+
+// ResolveConfig key/index to this is the ImageID which is allocated by the controller.
+// It will resolve container image tag to sha256
```

Comment:
Comments are out of date. Ditto for Status

Reply:
Updated the comments.